_data/egyedia/egyedia.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Attila L. Egyedi
+institution: Georgia Tech
+profile_pic: atti.png
+quote: Give me six hours to chop down a tree and I will spend the first four sharpening the ax. – Lincoln
```

Comment:
Change em dash – to single dash - as it is a non-ascii character.

Reply:
Thank you for the detailed error report, I fixed it!